scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -789,7 +888,85 @@
         [filter_second]
             side=1
         [/filter_second]
-        {MSG_narrator _"How did you manage to kill Bor Cryne? This shouldn't happen... it means the level is too easy. Mail adanilecki @ cs put poznan pl with your savegame and replay for review!"}
+#ifdef DEBUG_MODE
+        #po: Bor Cryne shouldn't die either way, whether debug mode is on or not, so the fact that we recognize debug mode specially here is just an easter egg:
+        {MSG_narrator _"Using debug mode to kill Bor Cryne is cheating!"}
+        [endlevel]
+            result=defeat
+        [/endlevel]
+#else
+        {MSG_narrator _"How did you manage to kill Bor Cryne? This shouldn't happen... it means the level is too easy. Stop what you are doing and mail adanilecki @ cs put poznan pl with your savegame and replay for review!"}
+        [kill]
```

Comment:
Same thing with this whole block below, I don't really like it. The about games is people should be able to play the way they like. You can insert a snide meta comment but actually having it affect gameplay is tacky. I think I'd rather not add this.

Reply:
the whole thing? I'll get rid of the call to `[kill]` and the ending of level in defeat, but can we at least keep the part where Bor Cryne gets healed? Because he really isn't supposed to die...

Reply:
Well I got rid of the call to `[kill]` and the ending of the level in defeat in 61ba872 at least